### ISSUE TITLE
Database update error

### ISSUE THREAD

**Issue by C-Wal** [dotnet-architecture/eShopOnWeb#843]
I am getting the following when running src\Web>dotnet ef database update --context catalogcontext

The database 'Microsoft.eShopOnWeb.CatalogDb' cannot be opened because it is version 904. This server supports version 852 and earlier. A downgrade path is not supported.

I am using VS 2022 and Microsoft SQL Server 2016 (SP1).  All I have changed since cloning the source is the connection strings to point to my development localdb instance.  The databases have been created successfully but on launch the app says migrations still need to be applied.

**Comment by C-Wal** [comment]
Yes I'm using localdb, though my instance has a different name, but I've changed the connection strings and it did create the databases and schemas.  My localdb is a SQL Server Express 2016 SP1 instance.

I have run all of the commands you stated but I still get a message when the app launches that migrations are required.  I tried running 'update-database -Context catalogcontext' and it said it was successful, but still the app said migrations were required so I tried the suggested 'dotnet ef database update' which doesn't work as there is more than one context, so I instead ran 'dotnet ef database update --context catalogcontext' which gave the error I provided.
